tests/test_app/test_logging.py

```diff
@@ -1,69 +1,122 @@
-import json
-
-import pytest
 import logging
 
-from tests.test_app.testing_app import app, db
-from tests.test_app.models import Bot
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy_logger.logger import Logger
+from dotenv import find_dotenv, load_dotenv
+import os
+
+from .models import Bot, Base
+
+
+load_dotenv(find_dotenv(".env.dev"))
+
 
+def make_engine():
```

Comment:
hmm returns session but the name `make_engine`

Reply:
I would still use pytest and made it a fixture. you should mark it as a dev dependency, so when someone installs your library with pip, they don't install pytest. rather someone can clone your repo and install deps from requirements.txt and run tests